server/planning/events/events_schema.py

```diff
@@ -195,7 +195,24 @@
         "mapping": {"properties": {"qcode": not_analyzed, "name": not_analyzed}},
     },
     # Content metadata
-    "subject": metadata_schema["subject"],
+    "subject": {
+        "type": "list",
+        "mapping": {
+            "type": "nested",
+            "include_in_parent": True,
+            "dynamic": False,
+            "properties": {
+                "qcode": string_with_analyzer,
+                "name": string_with_analyzer,
```

Comment:
this would not allow us to create aggregations, so we might need to have it indexed 2x